Enterprise-CMCS/macpro-onemac
# OY2 23605: 1915b Waiver Action Cards

Story: https://qmacbis.atlassian.net/browse/OY2-23605
Endpoint: See github-actions bot comment

### Details
Product team would like to use Waiver Authority to switch between required attachment types.  The best way to do this in our current form system is to pull Waiver Authority out as a form choice and use each config to separate the form requirements.  This PR creates the new card choices/path to arrive at the new forms successfully.

### Changes
- Added the new routes (including forms) to the route table
- Added the new triage routes (not including forms) to the React router widget
- Updated triage choices with the new texts, cards and paths
- Modified the ChoiceItem component and added a "Strong Text" line to handle the new bolded/italicized second line
- Updated styles for the new ChoiceItem strong text feature.

### Implementation Notes
I felt that it would be better to completely separate the forms based on waiver authority, rather than have one authority use the old forms, then the other is a "new type." But, I recognize this is easy to do with the triage and route tables and may not be the most efficient way to manage the forms side.  I feel like whomever does the form story is free to change this as they need.

### Test Plan
1. Login as a state submitter
2. Navigate to Dashboard
3. Click "New Submission" button
4. Verify the cards match the figma path (not the story, see comment in story)
5. Understand that one of the figma cards has cut-n-paste errors, extra (4)
